creator-node/package-lock.json

```diff
@@ -5849,9 +5849,9 @@
           }
         },
         "debug": {
-          "version": "4.3.1",
```

Comment:
are these package-lock changes important/relevant? I don't see any package.json updates in this PR either

Reply:
not important or relevant, but totally fine to have - if you npm install and any dependencies have newer versions that match the package.json semver spec then the package-lock will be updated
p normal, but i'll remove for now